sklearn/utils/multiclass.py

```diff
@@ -227,7 +228,7 @@ def type_of_target(y):
     >>> type_of_target(np.array([[1, 2], [3, 1]]))
     'multiclass-multioutput'
     >>> type_of_target([[1, 2]])
-    'multiclass-multioutput'
+    'multilabel-indicator'
```

Comment:
question: is this a bugfix?

Reply:
It is more-or-less a bug fix. Either way, type_of_target is expected to return "the most specific type that can be inferred". Multilabel is a specific subtype of multiclass-multioutput. So this is correct